app/views/catalog/_index_items_latest_received.html.erb

```diff
@@ -2,7 +2,7 @@
 <% unless lr_text.blank? %>
 <div class="<%= latest_received_wrapper_class %>">
       <dl>
-        <dt><span class='latest-received-badge badge badge-info'><%= t('trln_argon.show.badge.latest_received') %></span></dt>
+        <dt><span class='latest-received-badge badge bg-info'><%= t('trln_argon.show.badge.latest_received') %></span></dt>
```

Comment:
I'd just leave the `bg-info` out since it is having the effect of making those badges light blue rather than light gray. Looks like for whatever reason our BL7 UI had used `badge-info` but then used CSS to turn the badges back into gray.

Reply:
Removed.